src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,11 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: 'Under construction',
+  reducers: {
+    // No additional reducers needed
+  },
```

Comment:
- [ ] As the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions_v2.md#project-requirements) suggest, It would be much better to ensure that the initial state is initialized to the empty array of categories. In addition, I have noticed that you are missing a reducer that checks the status. Kindly implement this and ensure that it returns `Under construction`. ☺️